test/cql-pytest/test_tablets.py

```diff
@@ -201,3 +201,16 @@ def test_tablets_are_dropped_when_dropping_index(cql, test_keyspace, drop_index)
         except:
             pass
         raise e
+
+
+# Test that a LWT query fails with a tablets-enabled table.
+# See issue #18068
+def test_lwt_support_with_tablets(cql, test_keyspace, skip_without_tablets):
```

Comment:
Good to test this, but please state more clearly that this is a temporary situation that LWT fails with tablets, and when 18068 is solved, we need to delete this test.